save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/ContestController.kt

```diff
@@ -42,7 +61,18 @@ internal class ContestController(
      * @return contest with name [contestName]
      */
     @GetMapping("/{contestName}")
+    @RequiresAuthorizationSourceHeader
     @PreAuthorize("permitAll()")
+    @Operation(
+        method = "GET",
+        summary = "Get contest by name.",
```

Comment:
Do we really need both `summary` and `description`, while they are the same?
Isn't enough only one of them?

Reply:
`summary` is said to be less than 120 characters, I am not sure, but it might be displayed in a different way. Moreover it looks prettier because we have three arguments in every `@Operation`

Reply:
![screenshot](https://user-images.githubusercontent.com/35039155/180810470-887449c0-7e37-45ec-9b0e-1b9037c94127.png)
Take a look here: the endpoint at the top has only `description` set and the endpoint at the bottom has both of them set.